nginx/nginx.go

```diff
@@ -30,6 +30,12 @@ const (
 	metricsUpdateInterval = time.Second * 10
 )
 
+// IngressPortConf configuration
+type IngressPortConf struct {
```

Comment:
Avoid smurf naming, just call the struct "PortConf". Or simply "Port", so it will be `nginx.Port` when used elsewhere.